From 244b1eb7d87e7df7b393cad508b059a95ac27f0a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:14 -0400
Subject: [redhat] kabi: Adding symbol __pagevec_release (mm/swap.c)

Message-id: <087e577c8656768f8797de1ecc9993edef0acf01.1410510773.git.poros@redhat.com>
Patchwork-id: 95905
O-Subject: [PATCH RHEL7.1 014/233] kabi: Adding symbol __pagevec_release (mm/swap.c)
Bugzilla: 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Release pages back to the pagecache.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__pagevec_release
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__pagevec_release
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__pagevec_release

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__pagevec_release b/redhat/kabi/kabi-module/kabi_ppc64/__pagevec_release
new file mode 100644
index 0000000..759a7a1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__pagevec_release
@@ -0,0 +1,2 @@
+#1-
+0xd93502ca __pagevec_release vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__pagevec_release b/redhat/kabi/kabi-module/kabi_s390x/__pagevec_release
new file mode 100644
index 0000000..86c58ca
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__pagevec_release
@@ -0,0 +1,2 @@
+#1-
+0x4dbe9e2d __pagevec_release vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__pagevec_release b/redhat/kabi/kabi-module/kabi_x86_64/__pagevec_release
new file mode 100644
index 0000000..edd126e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__pagevec_release
@@ -0,0 +1,2 @@
+#1-
+0x3f614336 __pagevec_release vmlinux EXPORT_SYMBOL
-- 
1.7.1